logs/fluent-bit/k8s-manifest/README.md

```diff
@@ -28,19 +28,19 @@ Note: the configmap name is important and is being used by the daemonSet.
 
 change 'ENDPOINT' according to your logs endpoint from the table below.
 
-And apply it:
+And apply it to the monitoring namespace:
```

Comment:
Why we mentioned is mandatory?

Reply:
The role binding is applied to the monitoring namespace within the RBAC manifest.